### PR TITLE
Add instructions to migrate to Pyth from Chainlink

### PR DESCRIPTION
**Pull request by jayantk**
Add a how-to guide explaining how to use Pyth as a drop-in replacement for chainlink compatible apps.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ const permanentRedirectArray = [
     "/price-feeds/pythnet-price-feeds/best-practices",
     "/price-feeds/best-practices",
   ],
+  [
+    "/price-feeds/pythnet-price-feeds/pull-updates",
+    "/price-feeds/pull-updates",
+  ],
```
Comment (jayantk) [inline next.config.js:61]: drive-by: we moved this page and forgot a redirect. There are a couple inbound links to the old URL.

File: pages/price-feeds/migrate-an-app-to-pyth/chainlink.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+```text copy
+@pythnetwork/pyth-sdk-solidity/=node_modules/@pythnetwork/pyth-sdk-solidity
+```
+
+## Deploy Adapter Contract
+
+Chainlink-compatible applications can deploy and use `PythAggregatorV3` in `@pythnetwork/pyth-sdk-solidity` as a replacement for their Chainlink price feeds.
+`PythAggregatorV3` is an adapter contract that wraps the Pyth contract and implements Chainlink's `AggregatorV3Interface`.
+
+One important difference between Pyth and Chainlink is that the Pyth contract holds data for all price feeds, whereas Chainlink has separate instances of `AggregatorV3Interface` for each feed.
```
Comment (aditya520) [inline pages/price-feeds/migrate-an-app-to-pyth/chainlink.md:44]: This narrative feels better.

"Pyth contract holds data for all price feeds, in comparison to Chainlink, which has separate instances of AggregatorV3Interface` for every feed.

**Review by aditya520 — CHANGES_REQUESTED** [review]

**Review by aditya520 — APPROVED** [review]
